plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -361,6 +367,7 @@ public void widgetSelected(SelectionEvent e) {
       @Override
       public void run() {
         refreshProjectsForSelectedCredential();
+        refreshProjectsButton.setEnabled(accountSelector.getSelectedCredential() != null);
```

Comment:
Does this `refreshProjectButton` need to have its enablement state initially set?